setup.py

```diff
@@ -80,6 +85,7 @@ def find_version(*file_paths):
         "License :: OSI Approved :: BSD License",
     ],
     extras_require={
-        "dev": dev_requires,
-    }
+            "dev": dev_requires,
+            "http_server": http_server_requires,
```

Comment:
Is this name OK? Should I change any docs?

Reply:
yes I am fine with that